server/models/document.js

```diff
@@ -0,0 +1,47 @@
+
+/**
+ * @description define Document database model
+ * @param {Object} sequelize - sequelize object
+ * @param {Object} DataTypes - data types
+ * @returns {Object} - instance of Document database model
+ */
+module.exports = (sequelize, DataTypes) => {
+  const Document = sequelize.define('Document', {
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true,
+      validate: {
+        notEmpty: true,
+        len: [10, 60]
+      }
+    },
+    content: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+      validate: {
+        notEmpty: true
+      }
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    access: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      defaultValue: 'public',
+      validate: {
+        isIn: [['role', 'private', 'public']]
+      }
+    }
+  });
+
+  Document.associate = (models) => {
+    Document.belongsTo(models.User, {
+      foreignKey: 'userId',
+      onDelete: 'CASCADE',
```

Comment:
`onDelete: 'CASCADE'` is quite destructive and can lead to valuable documents being deleted automatically when the owner's account is deleted. You may want to consider less destructive alternatives.